src/github/dto/update.practices.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UpdatePracticesDto {
+  @ApiProperty({
+    description: 'Practices can be strings or a single paragraph.',
+    example: ['practice1', 'practice2'],
+  })
+  @IsNotEmpty()
+  practices: string[];
```

Comment:
PullProofAI

The `IsNotEmpty` decorator is applied directly to an array, which might not work as intended. It should be updated to validate the array elements individually.

```diff
-   @IsNotEmpty()
+   @ArrayNotEmpty()
```

### Note
Ensure that the `ArrayNotEmpty` validator is imported from `class-validator` if it's not already imported in the file.


<!-- This is an auto-generated comment by PullProofAI -->